src/components/EventDetails.scss

```diff
@@ -54,13 +54,19 @@
     }
   }
 }
-.time-icon,
+
+.column .event-icon {
+  margin-top: 12px;
+}
+
+.event-icon,
 .location-icon,
-.people-icon {
+.people-icon,
+.time-icon {
   height: 25px !important;
-  width: 25px !important;
   margin-right: 10px;
   overflow: visible !important;
+  width: 25px !important;
```

Comment:
Not really related to this PR, but I think we should never use `!important` because it makes it hard for developers to override app styling.

Reply:
@gcorne, I'll remove all instances of `!important` in a separate PR.